src/App.js

```diff
@@ -0,0 +1,11 @@
+import './styles/App.css';
+import React from 'react';
+import Router from './routes/routes';
+
+const App = () => (
+  <>
```

Comment:
- [ ] OPTIONAL: If I reload the page, all the information disappears from all sections.

The only way to display the information again is to logout and login again.  I suggest you look into this issue.